src/sui/themes/rtd-site/globals/site.overrides

```diff
@@ -23,8 +23,9 @@
 /*******************************
              Footer
 *******************************/
-footer {
+#footer {
   color: @footerColor;
+  margin-top: 0;;
```

Comment:
```suggestion
  margin-top: 0;
```

Reply:
VS Code automatically adds the same coma at the end that I already added on automatic mode...  :sigh:
Guess I could change this but decided to try and get used to it.